pkg/system/phase2_creating.go

```diff
@@ -258,6 +258,13 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 					if r.OAuthEndpoints != nil {
 						c.Env[j].Value = r.OAuthEndpoints.TokenEndpoint
 					}
+
+				case "NOOBAA_DISABLE_COMPRESSION":
```

Comment:
I know it's already implemented like that in noobaa-core but it bugs me anyway.
I think that for env vars its clearer and less error prone if it doesn't contain double negative (disabled=false)